fdbserver/ptxn/test/TestResolver.actor.cpp

```diff
@@ -55,20 +67,36 @@ std::vector<ResolveTransactionBatchRequest> makeTxnBatch(Version prevVersion, Ve
 	return batch;
 }
 
+// Returns the index in "batches", that item's version is "v".
+int findBatch(std::vector<ResolveTransactionBatchRequest> batches, Version v) {
+	for (int i = 0; i < batches.size(); i++) {
+		if (batches[i].version == v)
+			return i;
+	}
+	ASSERT(false);
+	return -1;
+}
+
 } // anonymous namespace
 
 TEST_CASE("fdbserver/ptxn/test/resolver") {
```

Comment:
I think we should add a comment describing what steps the test takes and what is checks.